src/NzbDrone.Core/Datastore/Migration/190_update_awesome_hd_link.cs

```diff
@@ -0,0 +1,15 @@
+using FluentMigrator;
+using Newtonsoft.Json.Linq;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(190)]
+    public class update_awesome_hd_link : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("UPDATE Indexers SET Settings = Replace(Settings, 'https://awesome-hd.net', 'https://awesome-hd.club') WHERE Implementation = 'AwesomeHD';");
```

Comment:
You are replacing `https://awesome-hd.net`, but the default was `BaseUrl = "https://awesome-hd.me";`... do we need to replace both?